src/MetaCompilation.Analyzers/MetaCompilation.Analyzers.sarif

```diff
@@ -4,7 +4,10 @@
   "runs": [
     {
       "tool": {
-        "name": "Microsoft Code Analysis",
+        "name": "MetaCompilation.Analyzers",
+        "version": "42.42.42.42",
```

Comment:
So I am afraid this can introduce noise. The version here indicates it is a debug version of the assembly, but the package eventually has the release version, and we don't auto-update the generated SARIF files on the build server for signed builds.

Reply:
I think a better option might be to pass in the version from Version.Props define [here](https://github.com/dotnet/roslyn-analyzers/blob/master/eng/Versions.props#L5) as a command line argument to the tool and use that. The file will only need to be updated everytime we bump the VersionPrefix after each release.

Reply:
But that was the version of the assembly that generated the info. If it's not the right assembly, who can guarantee that it's the same thing that will be packaged an deployed?

Reply:
Updated!